website/src/Components/PageComponents/TournamentsList.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import Tournament from './Tournament'
+
+class TournamentsList extends Component{
+	//will take in a js objects created from server data and render them in a table
+	constructor(){
+		super();
+		this.state={
+			tournaments: [
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			{	id: 0, description: 'a sample tournament', creatorName: 'sample creator', address: '1234 Sample Street',},
+			],
+		}
+	}
+
+	renderTournaments(){
```

Comment:
Why not have this function be render since render does nothing but just call this?